Sources/Player/PlaybackEngine/Internal/AudioSessionMonitors/AudioSessionRouteChangeMonitor.swift

```diff
@@ -54,6 +56,7 @@
 			      let asset = playerItem.asset,
 			      let configuration
 			else {
+				PlayerWorld.logger?.log(loggable: PlayerLoggable.changeMonitorUpdateVolumeWithoutRequiredData)
```

Comment:
We will log things even if we are not playing anything / nothing is loaded. Is that ok, or unecessary?

Reply:
I wanted to add logs in the monitors since I'm investigating an issue and would like to see if any change in the audio session is not being handled properly which could be causing the issue. I'd rather keep it at least during the on going investigation.